### ISSUE TITLE
feature request, chat gpt chat after google search 

### ISSUE THREAD

**Issue by jtagaca** [wong2/chatgpt-google-extension#242]
Just saw the new bing chat feature and I think that you would benefit well if you add a feature to your extension where chat gpt gives some context after your google search but you are able to chat with chat gpt in case you want to ask more questions. 

This feature is like what the bing chat is, but you will be the first to do this for chrome!!

**Comment by ravensergio** [comment]
Yes, a "continue conversation" would be great, maybe when u click continue it opens the extension window , or another ChatGPT tab and continue. In that way maybe the "delete history" would not trigger, and that's fine also.

**Comment by M-kasinski** [comment]
I made this with the fork of this product here : https://chrome.google.com/webstore/detail/chatgpt-online-for-google/llkfghafhndbplcfpcbnnkncbgmpjphf?hl=fr&authuser=0

**Comment by wong2** [comment]
@M-kasinski Have you released the source code? It is required by GPL-3

**Comment by M-kasinski** [comment]
@wong2 yes I published under the same license

**Comment by jtagaca** [comment]
@M-kasinski your extension doesn't allow chatting in the google search, that was the main point of this feature request.

**Comment by BlackHoleExelion** [comment]
Good point. A desired feature to me as well.

**Comment by dukesun99** [comment]
Does anyone know how new Bing actually handles such long context beyond 4k tokens? This might be the tricky part of web integration...
